webrtc.html

```diff
@@ -5345,32 +5363,38 @@ <h4>Processing Remote MediaStreamTracks</h4>
         process the removal of a remote track</dfn> for
         an incoming media description <span data-jsep=
         "applying-a-remote-desc">[[!JSEP]]</span> given
-        <code>RTCRtpTransceiver</code> <var>transceiver</var> and
-        <var>muteTracks</var>, the user agent MUST run the following steps:</p>
+        <code>RTCRtpTransceiver</code> <var>transceiver</var>,
+        <var>removeList</var>, and <var>muteTracks</var>, the user agent MUST
+        run the following steps:</p>
         <ol>
           <li>
             <p>Let <var>receiver</var> be <var>transceiver</var>'s
             <a>[[\Receiver]]</a>.
             </p>
           </li>
           <li>
-            <p>Let <var>track</var> be <var>receiver</var>'s
-            <a>[[\ReceiverTrack]]</a>.
+            <p>Let <var>msids</var> and <var>addList</var> be empty lists.
             </p>
           </li>
           <li>
-            <p><a>Set the associated remote streams</a> for the media
-            description, given <var>receiver</var> and an empty
-            list.</p>
+            <p><a>Set the associated remote streams</a>, given
+            <var>receiver</var>, <var>msids</var>, <var>addList</var>, and
+            <var>removeList</var>.</p>
```

Comment:
Since adding only adds to addList and remove only adds to removeList, I think it would be clearer not to have a separate section for "Set the associated remote streams" now that the firing of events has been moved to above.

Reply:
Nevermind it's also creating the media stream and other stuff, keep it.